seed/models/building_file.py

```diff
@@ -180,7 +180,8 @@ def process(self, organization_id, cycle, property_view=None):
             implementation_status = m['implementation_status'] if m.get('implementation_status') else 'Proposed'
             application_scale = m['application_scale_of_application'] if m.get('application_scale_of_application') else PropertyMeasure.SCALE_ENTIRE_FACILITY
             category_affected = m['system_category_affected'] if m.get('system_category_affected') else PropertyMeasure.CATEGORY_OTHER
-            recommended = str(m.get('recommended', 'false')).lower() == 'true'
+            # for some reason this is returning none if the field is empty. So none and true should both be true.
+            recommended = str(m.get('recommended', 'true')).lower() in ['true', 'none']
```

Comment:
just for context, we turn bsync into models in ~two steps:
1. parse bsync into some structure
2. turn structure into db models

for (1) we use `None` to signify we failed to parse the value from BSync (ie the XPath mapping returns no nodes / content), deferring how to handle that to the db/model level (ie here)

Reply:
gotcha!